src/polyfills.ts

```diff
@@ -77,7 +77,7 @@
 /***************************************************************************************************
  * Zone JS is required by default for Angular itself.
  */
-import 'zone.js/dist/zone' // Included with Angular CLI.
+import 'zone.js/dist/zone.js' // Included with Angular CLI.
```

Comment:
fllowing the suggestion at https://ckeditor.com/docs/ckeditor5/latest/builds/guides/integration/frameworks/angular.html#zonejs to fix unitest